packages/react-scripts/config/webpackDevServer.config.js

```diff
@@ -8,14 +8,14 @@
 // @remove-on-eject-end
 'use strict';
 
+const fs = require('fs');
 const errorOverlayMiddleware = require('react-dev-utils/errorOverlayMiddleware');
 const evalSourceMapMiddleware = require('react-dev-utils/evalSourceMapMiddleware');
 const noopServiceWorkerMiddleware = require('react-dev-utils/noopServiceWorkerMiddleware');
 const ignoredFiles = require('react-dev-utils/ignoredFiles');
 const paths = require('./paths');
-const fs = require('fs');
+const https = require('./https');
```

Comment:
Perhaps `createHttpsConfig`/`getHttpsConfig`? Just so it's a little clearer. Name the file and import the same way though.